src/metabase/lib/js/metadata.cljs

```diff
@@ -0,0 +1,150 @@
+(ns metabase.lib.js.metadata
```

Comment:
JavaScript implementation of a `MetadataProvider` that works with the metadata available to `Question.ts`